firestore/query.go

```diff
@@ -1666,15 +1637,15 @@ func (a *AggregationQuery) WithAvg(path string, alias string) *AggregationQuery
 
 // Get retrieves the aggregation query results from the service.
 func (a *AggregationQuery) Get(ctx context.Context) (AggregationResult, error) {
-	aro, err := a.GetWithOptions(ctx)
+	aro, err := a.GetResponse(ctx)
 	if aro != nil {
 		return aro.Result, err
 	}
 	return nil, err
 }
 
-// GetWithOptions runs the aggregation query results with the provided options
-func (a *AggregationQuery) GetWithOptions(ctx context.Context, opts ...RunOption) (aro *AggregationWithOptionsResult, err error) {
+// GetResponse runs the aggregation with the options provided in the query
+func (a *AggregationQuery) GetResponse(ctx context.Context) (aro *AggregationResponse, err error) {
```

Comment:
Here we already have `Get`, so `GetResponse` is more consistent than `Response`.